client/conn.go

```diff
@@ -408,3 +408,129 @@ func (c *Conn) exec(query string) (*Result, error) {
 
 	return c.readResult(false)
 }
+
+func (c *Conn) CapabilityString() string {
+	var caps []string
+	capability := c.capability
+	for i := 0; capability != 0; i++ {
+		field := uint32(1 << i)
+		if capability&field == 0 {
```

Comment:
not sure if you want [low-bit trick](https://www.quora.com/What-does-x-x-lowbit-do-in-C++)

Reply:
Oh, I'm not a big fan of unnecessary tricks :) I prefer more clear code.
This code executes only one time per connection, so I don't consider it a bottleneck.